src/DisplayPlug.vala

```diff
@@ -25,6 +26,8 @@ public class Display.Plug : Switchboard.Plug {
     private DisplaysView displays_view;
     private Gtk.Stack stack;
     private MirrorDisplay mirror_display;
+    private bool has_touchscreen = false;
+    private GLib.Settings rotation_lock_setting;
```

Comment:
The `Settings` object should be nullable like this: `private GLib.Settings? rotation_lock_setting`.